charts/hivemq-edge/templates/statefulset.yml

```
@@ -256,7 +278,17 @@ spec:
             items:
               - key: license.edgelic
                 path: license.edgelic
-      {{- end -}}
-      {{- if .Values.persistence.enabled -}}
-        {{ toYaml .Values.persistence.volume | nindent 8}}
-      {{- end -}}
+      {{- end }}
+  {{- if .Values.modules.persistence.enabled }}
```

Shouldn't this also be something like below, to match with the `volumeMount`?

```suggestion
  {{- if and .Values.modules.persistence.enabled .Values.license.enabled }}
```